ebmlite/util.py

```diff
@@ -474,8 +475,8 @@ def pprint(el: core.Element,
 #
 # ===========================================================================
 
-def printSchemata(paths: Optional[List[str]] = None,
-                  out: Union[str, IO] = sys.stdout,
+def printSchemata(paths: Optional[List[Union[str, Path]]] = None,
+                  out: Union[str, IO, Path] = sys.stdout,
```

Comment:
Minor stylistic preference: where you've added `Path` to an annotation that includes `str` and something non-`str` (here and elsewhere), could you put `Path` just after `str`? Both types serve the same purpose (a filename), but the non-`str` type works differently.